### ISSUE TITLE
Idea - Multi-Currency

### ISSUE THREAD

**Issue by RyanStankye** [butor/blackbird#49]
Note: High School student, This is a random idea I have had in my head for over half a year now, used for a bit, could be implemented with automation.

> Add checks for other currencies, taking advantage of more gaps.

I find that moving USD to BTC then to another currency (CAD/Euro) is safer for short term trades and leads to more profit (instead of keeping it and selling high). Keeping it in another currency helps reduce the risks of loss if a crash happens. (it adds more possibilities for profit by increasing surface area.)

tl;dr instead of running 5 bots on 5 different currencies,  have one check them all and increase the chances of profit, and or keeping money in a "stable" currency to flip.

Shoot me down if there is a major issue with implementation.

---

In case I didn't get my point across:

Profit off of gaps in other currencies along with bitcoin, with bitcoin as the switch currency.

**Comment by butor** [comment]
Hi @RyanStankye,

Thank you for your interest. Yes, adding exchanges in another currencies (CAD, EUR, CNY) will add more arbitrage opportunities since the currency conversion will add an extra layer of discrepancies between the prices. We actually briefly discussed this point on the BitcoinTalk forum ([link here](https://bitcointalk.org/index.php?topic=985660.msg12806512#msg12806512)). Is that what you had in mind?

Note that I don't see any issue with the implementation, except that the currency risk needs to be carefully hedged.

**Comment by RyanStankye** [comment]
@butor 

Yes, This is what I had in mind. Went through the forum, some great ideas being thrown around.
